src/main/java/marquez/resources/OwnerResource.java

```diff
@@ -25,4 +27,11 @@ public Response create(final Owner owner) {
     dao.insert(owner);
     return Response.created(buildURI(Owner.class, owner.getName())).build();
   }
+
+  @Path("{owner}")
+  @DELETE
+  @Consumes(APPLICATION_JSON)
```

Comment:
We can remove `@Consumes()` annotation given that no Json payload is provided along with the request.

Reply:
nice